app/controllers/movies_controller.rb

```diff
@@ -1,6 +1,24 @@
 class MoviesController < ApplicationController
   def index; end
 
+  def show
+    conn = Faraday.new(url: 'https://api.themoviedb.org') do |faraday|
+      faraday.headers['X-API-Key'] = ENV['MOVIEDB_API_KEY']
+    end
+    
+    movie = conn.get("/3/movie/#{params[:id]}?api_key=#{ENV['MOVIEDB_API_KEY']}&language=en-US")
+    @movie_details = JSON.parse(movie.body, symbolize_names: true)
+
+    @runtime = "#{@movie_details[:runtime] / 60}hrs, #{((@movie_details[:runtime].to_f / 60 - @movie_details[:runtime] / 60) * 60).to_i} minutes"
+
+    credits = conn.get("/3/movie/#{params[:id]}/credits?api_key=#{ENV['MOVIEDB_API_KEY']}")
+    credits_details = JSON.parse(credits.body, symbolize_names: true)
+    @actors = credits_details[:cast].take(10)
+
+    reviews = conn.get("/3/movie/#{params[:id]}/reviews?api_key=#{ENV['MOVIEDB_API_KEY']}&language=en-US&page=1")
+    @reviews_details = JSON.parse(reviews.body, symbolize_names: true)[:results]
+  end
```

Comment:
This method is actually one line too many too.  Maybe Runtime could be a helper method to both break up the length of the line and to get this method shorter?